src/agentscope/rag/llama_index_rag.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+"""
+This module is integrate the Llama index RAG model into our AgentScope package
+"""
+
+from typing import Any, Union, Type, Optional
+from pathlib import Path
+from llama_index.core.readers.base import BaseReader
+from llama_index.core.base.base_retriever import BaseRetriever
+from llama_index.core import VectorStoreIndex, SimpleDirectoryReader
+
+from agentscope.rag import RAGBase
+from agentscope.models import ModelWrapperBase
+
+
+class LlamaIndexRAG(RAGBase):
+    """
+    This class is a wrapper around the Llama index RAG.
+    """
+
+    def __init__(
```

Comment:
we also need an embedding model in LLamaIndexRAG, right?

Reply:
Indeed, how to convert a model wrapper in AgentScope into the LlamaIndex format remains a problem.

Reply:
> we also need an embedding model in LLamaIndexRAG, right?

Current version support passing embedding model of either `ModelWrapperBase` or `BaseEmbedding` in llama-index to this module.